src/Interface/Modules/Render/ViewScene.cc

```diff
@@ -522,10 +522,20 @@ void ViewSceneDialog::pullCameraLookAt()
   auto spire = mSpire.lock();
   if(!spire) return;
 
-  auto lookAt = toDoubleVector(state_->getValue(Modules::Render::ViewScene::CameraLookAt).toVector());
+  auto lookAt = pointFromString(state_->getValue(Modules::Render::ViewScene::CameraLookAt).toString());
   spire->setCameraLookAt(glm::vec3(lookAt[0], lookAt[1], lookAt[2]));
 }
 
+//--------------------------------------------------------------------------------------------------
+glm::quat ViewSceneDialog::stringToQuat(std::string &s)
+{
+  std::istringstream is(s);
+  double w, x, y, z;
+  char st;
+  is >> st >> w >> st >> x >> st >> y >> st >> z >> st;
```

Comment:
This is also a bit brittle when Python input is considered. Look into `boost::regex` to handle the intervening commas/spaces better. And add a unit test.